contractcourt/channel_arbitrator.go

```diff
@@ -965,10 +965,10 @@ func (c *ChannelArbitrator) stateStep(
 			// Ignore errors since logging force close info is not a
 			// critical part of the force close flow.
 			ChainTriggerForceClose(chainActions)(c)
-			fallthrough
+			nextState = StateBroadcastCommit
```

Comment:
Because we do not want this function called for https://github.com/lightningnetwork/lnd/pull/8072/files/c60ea993c2a31fe62c3e481cd40618455bf60afd..b22d72d44d5c71f732f15e5a796f5de7fb39a451#diff-a0b8064876b1b1d6085fa7ffdbfd38c81cb06c1ca3f34a08dbaacba203cda3ebL969-L970 on line 970 called for chain trigger force close. Chain action would be recorded twice in that case.